__init__.py

```diff
@@ -76,10 +78,29 @@ def _setup(self, force=False):
                     self.enable_fallback = \
                         self.settings.get('enable_fallback')
 
+                # Register tracker entities
+                self._list_tracker_entities()
+
     def _force_setup(self):
         self.log.debug('Creating a new HomeAssistant-Client')
         self._setup(True)
 
+    def _list_tracker_entities(self):
+        """List tracker entities.
+
+        Add them to entity file and registry it so
+        Padatious react only to known entities.
+        Should fix conflict with Where is skill.
+        """
+        types = ['device_tracker']
+        entities = self.ha_client.list_entities(types)
+
+        if entities:
+            entity_file = pth_join(self.root_dir, 'vocab', 'tracker.entity')
```

Comment:
I just realised you're writing to the Skills `root_dir` here. That's not recommended as that directory is not guaranteed to be writable by the Skill, and making any change to that directory will cause the Skill to reload.

I'd suggest switching this to be stored in the cache or persistent fs:
https://mycroft-ai.gitbook.io/docs/skill-development/skill-structure/filesystem

Reply:
Temporary Cache sounds good, will try.